addons/l10n_nl/migrations/3.3/post-migrate_update_taxes.py

```diff
@@ -0,0 +1,58 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import api, SUPERUSER_ID
+
+
+def migrate(cr, version):
+    env = api.Environment(cr, SUPERUSER_ID, {})
+    goods_taxes = env['account.tax'].search([('name', '=', 'Verkopen export binnen EU (producten)')], limit=1)
+    services_taxes = env['account.tax'].search([('name', '=', 'Verkopen export binnen EU (diensten)')], limit=1)
+
+    old_plus_tax_tag_id = env['account.account.tag'].search(env['account.account.tag']._get_tax_tags_domain('3bl (omzet)', 'nl', '+'), limit=1).id
```

Comment:
Just to be picky, but I would still use `_get_tax_tags()` and then filter the result on `tax_negate` :blush:

Reply:
Okey no problem I will do that :)